src/pages/learn/index.mdx

```diff
@@ -1,42 +1,46 @@
+import { Callout } from "nextra/components"
+
 # Introduction to GraphQL
 
-> Learn about GraphQL, how it works, and how to use it. Looking for documentation on how to build a GraphQL service?
-> There are libraries to help you implement GraphQL in [many different languages](/code). For an in-depth learning experience
-> with practical tutorials, see [the available training courses](/community/resources/training-courses).
+<p className="learn-subtitle">Learn about GraphQL, how it works, and how to use it</p>
+
+GraphQL is a query language for your API, and a server-side runtime for executing queries using a type system you define for your data. The [GraphQL specification](https://spec.graphql.org/) was open-sourced in 2015 and has since been implemented in a variety of programming languages. GraphQL isn't tied to any specific database or storage engine—it is backed by your existing code and data.
 
-GraphQL is a query language for your API, and a server-side runtime for executing queries using a type system you define for your
-data. GraphQL isn't tied to any specific database or storage engine and is instead backed by your existing code and data.
+<Callout type="info">If you're already familiar with GraphQL and would like to read documentation on how to build a GraphQL service, then there are libraries to help you implement GraphQL in [many different languages](/community/tools-and-libraries/?tags=server). There are also many libraries available that allow [client applications to query existing GraphQL APIs](/community/tools-and-libraries/?tags=client).</Callout>
 
-A GraphQL service is created by defining types and fields on those types, then providing functions for each field on each type.
-For example, a GraphQL service that tells you who the logged in user is (`me`) as well as that user's name might look
-like this:
+## Describe your API with a type system
+
+A GraphQL service is created by [defining types and their fields](/learn/schema/), and then writing a function for each field to [provide the required data](/learn/execution/). For example, a GraphQL service that tells you the name of a logged-in user might look like this:
 
 ```graphql
 type Query {
   me: User
 }
 
 type User {
-  id: ID
   name: String
 }
 ```
 
 Along with functions for each field on each type:
 
 ```js
-function Query_me(request) {
-  return request.auth.user
+// Provide data for the `me` field on the `Query` type
+function Query_me(obj, args, context, info) {
+  return context.request.auth.user
```

Comment:
I wasn't sure whether to put `query` or `rootValue` or `source` or `parent` or `_` here; I went with `query` in the end for consistency with `user` below.

```suggestion
function Query_me(query, args, context, info) {
  return context.request.auth.user
```